src/Microsoft.Extensions.Configuration.AzureAppConfiguration/KeyValueChange.cs

```diff
@@ -23,5 +23,7 @@ internal struct KeyValueChange
         public ConfigurationSetting Current { get; set; }
 
         public ConfigurationSetting Previous { get; set; }
+
+        public bool IsMultiKeyWatcherChange { get; set; }
```

Comment:
Yes that was my fault on the naming, replacing this with `IsWatchedSetting` because it was being used to determine whether this change was from being watched as a collection or as an individual setting.